ingest/template/descriptor.py

```diff
@@ -116,6 +117,7 @@ def __init__(self, json_data):
                 # identifiable properties
                 if property_name in IDENTIFIABLE_PROPERTIES:
                     child_property_descriptor.identifiable = True
+                    child_property_descriptor.external_reference = True
```

Comment:
The logic is actually, if property name is `uuid` , it'll be considered as external reference. So in the spreadsheet, you can have donor.uuid, cell_suspension.uuid, sequencing_file.uuid then that means it's referring to an entity already submitted before. It's used for linking to an existing entity or when specifying the updates in the spreadsheet.

Reply:
Done! Thank you for the explanation!